src/expr/src/scalar/func.rs

```diff
@@ -2814,6 +2826,7 @@ impl BinaryFunc {
 impl fmt::Display for BinaryFunc {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         match self {
+            BinaryFunc::UuidGenerateV5 => f.write_str("+"),
```

Comment:
Is `"+"` here correct? I would have thought that it was something like `"uuid_generate_v5"`.

Reply:
This should also be in its sorted spot.

Reply:
ah, good catch. Sorry missed that in cargo culting to get the build to pass. Fixing